src/models/methods/ledger.ts

```diff
@@ -0,0 +1,58 @@
+import { LedgerIndex } from "../common";
+import { LedgerEntry } from "../ledger";
+import { BaseRequest, BaseResponse } from "./baseMethod";
+
+export interface LedgerRequest extends BaseRequest {
+    command: "ledger"
+    ledger_hash?: string,
+    ledger_index?: LedgerIndex
+    full?: boolean
+    accounts?: boolean
+    transactions?: boolean
+    expand?: boolean
+    owner_funds?: boolean
+    binary?: boolean
+    queue?: boolean
+}
+
+interface Ledger {
+    account_hash: string
+    accountState?: LedgerEntry[]
+    close_flags: number
+    close_time: number
+    close_time_human: string
+    close_time_resolution: number
+    closed: boolean
+    ledger_hash: string
+    ledger_index: string
+    parent_close_time: number
+    parent_hash: string
+    total_coins: string
+    transaction_hash: string 
+    transactions?: any[] // TODO: Retype this once we have transaction types
+}
+
+interface LedgerQueueData {
+    account: string
+    // TODO: Retype tx once we have transaction types 
+    // TODO: Also include tx_blob as possible type: https://xrpl.org/ledger.html
+    // TODO: Also handle the special case where 'owner_funds: string' is a field of OfferCreate sometimes - https://xrpl.org/ledger.html#response-format
```

Comment:
These should all be under one TODO, so when searching for TODOs in the code they aren't displayed as separate TODOs